samples/disas/disas.cpp

```diff
@@ -407,6 +407,25 @@ int WINAPI WinMain(HINSTANCE hinst, HINSTANCE hprev, LPSTR lpszCmdLine, int nCmd
     (void)hinst;
     (void)lpszCmdLine;
     (void)nCmdShow;
+
+    // Bug report, but it works here.
+    // 07ff8`4b783054 49ba 70b3d93a d40fb998 mov r10,98B90FD43AD9B370h
+    //
+    {
+        static const UCHAR mov_r10_imm64[] = {0x49, 0xba, 1, 2, 3, 4, 5, 6, 7, 8 };
+
+        PVOID const after = DetourCopyInstructionX64(0, 0, const_cast<PUCHAR>(mov_r10_imm64), 0, 0);
+
+        if (after != &mov_r10_imm64 + 1)
+        {
+            printf("mov_r10_imm64 failed, expected:%p vs. got:%p\n", &mov_r10_imm64 + 1, after);
+            __debugbreak();

```

Comment:
The other tests seem to "return 1;" on failure to make the failure obvious.
Shall we follow that convention?